lib/tasks/locale.rake

```diff
@@ -298,6 +298,16 @@ namespace :locale do
         system "rmsgcat -o #{po} #{po_files[locale].join(' ')}"
       end
 
+      # create webpack file for including bootstrap-datepicker language packs
+      File.open(::ManageIQ::UI::Classic::Engine.root.join('app/javascript/packs/bootstrap-datepicker-languages.js'), "w+") do |f|
+        f.puts("// This is file is automatically generated by rake locale:po_to_json task")
```

Comment:
Suggest - This file is automatically generated by the rake task ‘locale:po_to_json’